### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by oriontvv**
This PR adds dependabot config

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    open-pull-requests-limit: 10
+  - package-ecosystem: "docker"
```
Comment (idanarye) [inline .github/dependabot.yml:8]: Is this necessary? I'm not using Docker here. Unless GitHub Actions uses it behind the scenes - but wouldn't that be handled by the `package-ecosystem: "github-actions"` section?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    directory: "/"
+    schedule:
+      interval: "monthly"
+  - package-ecosystem: "github-actions"
+    directory: "/"
```
Comment (idanarye) [inline .github/dependabot.yml:13]: I'm not familiar with dependabot (did not even know about its existence) but isn't the directory for GitHub Actions `.github/workflows`?

File: .github/dependabot.yml (hunk 1 above)
Comment (oriontvv) [reply .github/dependabot.yml:8]: Ok, I added common template - will remove docker section. Fyi, if you don't have any Dockerfiles - the job would just skip it

File: .github/dependabot.yml (hunk 2 above)
Comment (oriontvv) [reply .github/dependabot.yml:13]: Not really, it's the correct path

File: .github/dependabot.yml (hunk 1 above)
Comment (idanarye) [reply .github/dependabot.yml]: If it's a common template then leave it as is. No reason to deviate from the standards.

**Pull request merged**